x-pack/plugins/cases/server/saved_object_types/migrations/cases.ts

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
I moved the cases migrations to their own field under the new directory `migrations` because the `migrations` file is getting kind of large.

Reply:
Thanks for that!